src/main/java/net/minecraftforge/event/world/RaidEvent.java

```diff
@@ -0,0 +1,81 @@
+package net.minecraftforge.event.world;
+
+import net.minecraft.entity.monster.AbstractRaiderEntity;
+import net.minecraft.world.raid.Raid;
+import net.minecraftforge.eventbus.api.*;
+
+/**
+ *  RaidEvent is the base class of all Raid events.
+ */
+@Cancelable
+public class RaidEvent extends Event
+{
+   private final Raid raid;
+   
+   public RaidEvent(Raid raid)
+   {
+	  this.raid = raid;
+   }
+   
+   public Raid getRaid()
+   {
+	   return raid;
+   }
+   
+   /**
+    * This event is fired for every {@link Raid} tick. 
+    * This event is {@link Cancelable}.
+    * If this event is cancelled, each tick will be skipped.
```

Comment:
That is not true... If the event is cancelled _that tick_ will be skipped. Not each of them.